### DISCUSSION TITLE
[Feature] Group similar images using CLIP

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#2447] [Feature Request]
### The feature

The image embedding CLIP can be used for finding images that looks (or conceptually) similar. This will bring a lot of interesting use cases.

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by chriexpe** [comment]
Agreed, it could be just like when you search a image on the web, you open it and below there is "relatable content".
On Immich it might be interesting to, when you have a photo/video on full screen, scroll down the page (exactly like in "stories") where it lists other photo/videos that have similar tag embeddings.
Also it would fit great with this other feature request of "image interrogator" https://github.com/immich-app/immich/discussions/2519 that generates way more tags than the currently used "Clips".

**Comment by bo0tzz** [comment]
#21108 #22148
